sklearn/metrics/tests/test_common.py

```diff
@@ -1950,7 +1950,6 @@ def check_array_api_regression_metric_multioutput(
 
 
 def check_array_api_metric_pairwise(metric, array_namespace, device, dtype_name):
-
     X_np = np.array([[0.1, 0.2, 0.3], [0.4, 0.5, 0.6]], dtype=dtype_name)
```

Comment:
For the sake of keeping the diff of the PR easier to review, let's avoid such unrelated style changes.